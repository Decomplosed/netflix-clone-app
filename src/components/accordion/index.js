import React from 'react';

export default function Accordion({ children, ...restProps }) {
  return <Container></Container>;
}
