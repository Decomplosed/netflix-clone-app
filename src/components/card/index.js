import React from 'react';

export const FeatureContext = createContext();
