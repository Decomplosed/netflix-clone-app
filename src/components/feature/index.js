import React from 'react';
import { Container, Title, SubTitle } from './styles/feature';

export default function Feature({ children, ...restProps }) {
  return <Container></Container>
}
