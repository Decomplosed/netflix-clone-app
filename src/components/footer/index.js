import React from 'react';

export default function Footer({ children, ...restProps }) {
  return <Container {...restProps}></Container>;
}
