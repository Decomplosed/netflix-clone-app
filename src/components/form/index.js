import React from 'react';
import {
  Container,
  Base,
  Error,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form';

export default function Form({ children, ...restProps }) {
  return <Container {...restProps}>{children}</Container>;
}

Form.Base = function FormBase({ children, ...restProps }) {
  return <Base {...restProps}>{children}</Base>;
};

Form.Error = function FormError({ children, ...restProps }) {
  return <Error {...restProps}>{children}</Error>;
};

Form.Title = function FormTitle({ children, ...restProps }) {
  return <Title {...restProps}>{children}</Title>;
};
