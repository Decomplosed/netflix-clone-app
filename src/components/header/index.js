import React from 'react';

export default Header({bg = true, children, ...restProps}) {}