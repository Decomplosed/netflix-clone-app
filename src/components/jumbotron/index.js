import React from 'react';

export default Jumbotron = () => {
  return <p>I am jumbotron</p>;
};
