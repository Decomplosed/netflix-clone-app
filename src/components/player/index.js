import React, { useState, useContext, createContext } from 'react';
import ReactDOM from 'react-dom';
