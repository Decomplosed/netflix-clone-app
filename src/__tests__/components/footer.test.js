import React from 'react';
import { render } from '@testing-library/react';
import { Footer } from '../../components';

describe('<Footer />', () => {
  it('Renders the <Footer /> with populated data', () => {
    const { container, getByText } = render(
      <Footer>
        <Footer.Title></Footer.Title>
      </Footer>,
    );
  });
});
