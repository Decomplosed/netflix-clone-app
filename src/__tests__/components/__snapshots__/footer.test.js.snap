// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Renders the <Footer /> with populated data 1`] = `
<div
  class="footer__Container-v6r34u-0 kBHsfy"
>
  <p
    class="footer__Title-v6r34u-4 cqpRFo"
  >
    Questions? Contact us.
  </p>
  <div
    class="footer__Break-v6r34u-6 eEHPHH"
  />
  <div
    class="footer__Row-v6r34u-2 hgJQay"
  >
    <div
      class="footer__Column-v6r34u-1 fnHQoR"
    >
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        FAQ
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Investor Relations
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Ways to Watch
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Corporate Information
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Netflix Originals
      </a>
    </div>
    <div
      class="footer__Column-v6r34u-1 fnHQoR"
    >
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Help Centre
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Jobs
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Terms of Use
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Contact Us
      </a>
    </div>
    <div
      class="footer__Column-v6r34u-1 fnHQoR"
    >
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Account
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Redeem gift cards
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Privacy
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Speed Test
      </a>
    </div>
    <div
      class="footer__Column-v6r34u-1 fnHQoR"
    >
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Media Centre
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Buy gift cards
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Cookie Preferences
      </a>
      <a
        class="footer__Link-v6r34u-3 fjoZjt"
        href="#"
      >
        Legal Notices
      </a>
    </div>
  </div>
  <div
    class="footer__Break-v6r34u-6 eEHPHH"
  />
  <p
    class="footer__Text-v6r34u-5 kBOPMl"
  >
    Netflix United Kingdom
  </p>
</div>
`;
